src/model/RacingGame.js

```diff
@@ -0,0 +1,57 @@
+import { CAR } from '../constants/constants.js';
+import Car from './Car.js';
+
+export default class RacingGame {
+  constructor() {
+    this.cars = [];
+    this.round = CAR.INITIAL_VALUE;
+  }
+
+  moveCars() {
+    this.cars.forEach((car) => {
+      car.move();
+    });
+  }
+
+  set players(names) {
+    this.cars = names.map((name) => new Car(name));
+  }
+
+  goToNextTurn() {
+    this.moveCars();
+    this.round -= CAR.ONE_MOVE;
+  }
+
+  get winners() {
+    return this.findWinners();
+  }
+
+  reset() {
+    this.cars = [];
+    this.round = CAR.INITIAL_VALUE;
+  }
+
+  findMaxRecord() {
+    const maxRecord = this.cars.reduce((accumulator, current) => {
+      if (current.position > accumulator) {
+        accumulator = current.position;
+      }
+      return accumulator;
+    }, CAR.INITIAL_VALUE);
```

Comment:
요런거는 아래와같이 축약할 수 있겠죠?! (참고만 해주세요~!)

```suggestion
    const maxRecord = this.cars.reduce((accumulator, current) => Math.max(accumulator, current.position), CAR.INITIAL_VALUE);
```